test/unit/dashboard-view-model.spec.js

```diff
@@ -179,4 +179,29 @@ describe('DashboardViewModel', function () {
       expect(model.timeline).to.eql(expectedTimeline);
     });
   });
+
+  describe('shut down counter', function () {
+    [
+      {
+        name: 'should return the number of days remaining',
+        shutDownDate: moment(now).add(22, 'days'),
+        expectedDate: 'in 22 days',
+      },
+      {
+        name: 'should return today when shut down date is today',
+        shutDownDate: now,
+        expectedDate: 'today',
+      },
+      {
+        name: 'should return something when shut down date is in past',
```

Comment:
Would be worth checking, but I would say that Mel would want this to just say imminently rather than counting down

Reply:
The other suggested text was:

This version of 'Record your work' search is going to shut down on [DATE].

but both Mel and Sean preferred the count down.